### ISSUE TITLE
2 cleanup items in plot_logs function (futureproof warning, crash if no log.txt exists)

### ISSUE THREAD

**Issue by lessw2020** [facebookresearch/detr#172]
## Instructions To Reproduce the 🐛 Bug:
Upgraded my server to PyTorch 1.6 - begin receiving this error when using plot_logs:
'/home/ubuntu/detr/util/plot_utils.py:55: FutureWarning: The pandas.np module is deprecated and will be removed from pandas in a future version. Import numpy directly instead
  pd.np.stack(df.test_coco_eval_bbox.dropna().values)[:, 1]'

and if I run plot_logs before a logs.txt has been written (b/c I'm very eager to check on progress :) then get a crash instead of an error informing me no log.txt is available yet.

1. what changes you made (`git diff`) or what code you wrote
```
To future proof:
import numpy as np
remove pd from line 55 to become:
if field == 'mAP':
                coco_eval = pd.DataFrame(
                    **np.**stack(df.test_coco_eval_bbox.dropna().values)[:, 1]

2 - to handle no log file exists:
# check if first log file exists...it will not be present until after first log write
    fn = Path(logs[0]/log_name)
    
    if not fn.exists():
        print(f"-> missing {log_name} in first directory.  This is likely b/c you have not gotten to Epoch 1 in training.")
        print(f"--> file path: {fn}")
        return


```
2. what exact command you run:
3. what you observed (including __full logs__):
```
<put logs here>
```
4. please simplify the steps as much as possible so they do not require additional resources to
	 run, such as a private dataset.

## Expected behavior:

If there are no obvious error in "what you observed" provided above,
please tell us the expected behavior.

## Environment:

Provide your environment information using the following command:
```
python -m torch.utils.collect_env
```

**Comment by fmassa** [comment]
Can you send a PR with the proposed fixes?

**Comment by lessw2020** [comment]
Yes happy to, thanks for the feedback.

**Comment by fmassa** [comment]
Merged in #194 thanks @lessw2020 !
